game-core/src/main/java/games/strategy/triplea/ui/menubar/HelpMenu.java

```diff
@@ -238,13 +239,14 @@ public void windowOpened(final WindowEvent e) {
   private void addGameNotesMenu() {
     // allow the game developer to write notes that appear in the game
     // displays whatever is in the notes field in html
-    final String notesProperty = gameData.getProperties().get("notes", "");
-    if (notesProperty != null && notesProperty.trim().length() != 0) {
-      final String notes = LocalizeHtml.localizeImgLinksInHtml(notesProperty.trim());
+    final String trimmedNotes = gameData.getProperties().get("notes", "").trim();
```

Comment:
Could this cause a NPE?

Reply:
No, because we pass a default parameter.